applications/ChimeraApplication/custom_utilities/hole_cutting_utility.cpp

```diff
@@ -0,0 +1,438 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Authors:        Aditya Ghantasala, https://github.com/adityaghantasala
+// 					Navaneeth K Narayanan
+//					Rishith Ellath Meethal
+//
+
+#include "hole_cutting_utility.h"
```

Comment:
I think you could improve the efficiency of this utility
esp considering this is needed all the time

Reply:
I am happy to discuss this with you and improve in future.